cmd/kubeapps-apis/server/packages.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright © 2021 VMware
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package server
+
+import (
+	"context"
+
+	packages "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/core/packages/v1alpha1"
+)
+
+// packagesServer implements the API defined in proto/kubeappsapis/core/packages/v1alpha1/packages.proto
+type packagesServer struct {
+	packages.UnimplementedPackagesServiceServer
```

Comment:
This is the specially generated struct which we embed as always, we're just not forced to do so with the `require_unimplemented_servers=false`, and since the required interface is then generated *without* the private method, plugin servers are able to satisfy the interface as long as they implement the actual functions (eg. `GetPackageRepositories`, `GetAvailablePackages`, ... whatever else we add).